### ISSUE TITLE
Polish IBAN doesn't work

### ISSUE THREAD

**Issue by ignissak** [mdomke/schwifty#105]
Hey everyone.

I am currently working with schwifty v2022.7.0 and I am trying to parse polish IBAN `PL20 1050 1214 1000 0090 3236 2619`. However, schwifty somehow does not resolve this IBAN and I cannot get a bank name nor BIC. This IBAN is valid, you can check it on other website apps. So I wonder where could be the problem because I could find BIC in the polish registries.

**Comment by mdomke** [comment]
Retrieving the BIC should actually work
```
>>> from schwifty import IBAN
>>> IBAN("PL20 1050 1214 1000 0090 3236 2619").bic
<BIC=INGBPLPWXXX>
```
but calling `IBAN(...).bank` actually returns `None`. This is because the Polish banks don't have a bank code, but use the branch code as sort code. I will create a fix for this in a minute.

**Comment by mdomke** [comment]
Fixed in version `2022.7.1`
